docs/src/tutorials/child-device-config-mngmnt.md

```diff
@@ -0,0 +1,297 @@
+
+In this tutorial 127.0.0.1. is used.
+
+##  Prerequisite
+
+To follow this tutorial, you only need the following:
+
+- A [Cumulocity IoT](https://www.softwareag.cloud/site/product/cumulocity-iot.html) tenant.
+- Installed the latest version of thin-edge.io. To this, you can follow [this tutorial](https://thin-edge.github.io/thin-edge.io/html/tutorials/getting-started.html#Step-6-Manage-configuration-files), which will also give you some deeper insights in the various capabilities of thin-edge.io, like configuration management.
+- A Raspberry Pi (3 or 4) with Raspian installed, configured and connected to the Cumulocity tenant. For other boards and OS'es have a look [here](https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/supported-platforms.md)
+- Updated device: `$ sudo apt-get update && sudo apt-get upgrade`
+
+
+## Steps
+
+This tutorial is divided into 3 main steps.
+
+[Step 1 Startup / Bootstrap of the child device](#Startup-/-Bootstrap-of-the-child-device).
+
+In this step a child device with its supported configuration list will be created and will be sent to thin-edge.io during the startup/bootstrap phase of the child device agent.
+
+[Step 2 Handle config snapshot request from thin-edge.io](#handle-config-snapshot-request-from-thin-edge.io)
+
+In this step configuration files from a child device will be requested to make them visible in Cumulocity IoT.
+
+[Step 3 Handle config update request from thin-edge.io](#handle-config-update-request-from-thin-edge.io)
+
+In this configuration files from a child device will be updated.
+
+
+### Step 1 Startup / Bootstrap of the child device
+
+In this step a child device with its supported configuration list will be created and will be sent to thin-edge.io during the startup/bootstrap phase of the child device agent.
+
+Therefore some steps have to be taken
+
+1. Create a `c8y-configuration-plugin.toml` this file consists of the supported configuration list i.e. a list of configuration files.
+
+Note: this is a separate file and not the already exsisting `c8y-configuration-plugin.toml` file which is used for configuration management of thin-edge.io.
+
+The child device agent needs to capture the list of configuration files that needs be managed from the cloud in a `c8y-configuration-plugin.toml` file in the same format as specified in the [configuration management documentation](https://thin-edge.github.io/thin-edge.io/html/howto-guides/025_config_management_plugin.html) as follows:
```

Comment:
Could you combine this line with the bullet point.1 above, to avoid redundancy.

Reply:
done